app/code/community/Oyst/OneClick/etc/system.xml

```diff
@@ -616,6 +616,27 @@
                         </time>
                     </fields>
                 </order_delay>
+
+                <checkout_cart_button translate="label">
```

Comment:
Change for `customization_button_checkout_cart` like that all new customization fields will move here.